src/pages/bridge/Send/SendAmountSelectorCard.tsx

```diff
@@ -223,7 +223,13 @@ const SendAmountSelectorCard: FC<Props> = props => {
               </button>
             ) : null} */}
           </div>
-        ) : null}
+        ) : (
```

Comment:
Could we refrain from displaying "Balance: 0" if the wallet is not connected?

Reply:
agree with you, just keep displaying nothing when disconnected